plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -202,7 +203,9 @@ class JvmAbiOutputExtension(
                 val next = stack.removeLast()
                 add(next)
                 // Classes form a tree by nesting, so none of the children have been visited yet.
-                innerClassesByOuterName[next]?.mapNotNullTo(stack) { it.name.takeIf(abiClassInfos::contains) }
+                innerClassesByOuterName[next]?.mapNotNullTo(stack) { info ->
+                    info.name.takeIf { abiClassInfos[it] != AbiClassInfo.Deleted }
```

Comment:
I would prefer to use `getValue` here as well, instead of `get`

Reply:
Fixed in https://github.com/JetBrains/kotlin/commit/2dfd6ad4077784846eb52cbb0e81abebe29975c8

Reply:
Thank you for the correction. Fail fast.